roles/ldapserver/tasks/main.yml

```diff
@@ -39,3 +39,204 @@
   loop:
     - people
     - groups
+
+- name: Place TLS certs
+  ansible.builtin.copy:
+    content: "{{ ldap_tls_cert }}"
```

Comment:
I think you should consider if it's smarter to store the cert as a file instead of as variable so its both easier to generate and the variables file is easier to read.

Reply:
but I would not want to commit the test cert or key that I have been using.   Or change that content: to src: and ldap_tls_cert to a ldap_tls_cert_file ?

Reply:
I changed this to ldap_tls_cert_file.